### ISSUE TITLE
Add problem statements as comments in the start

### ISSUE THREAD

**Issue by debodirno** [haoel/leetcode#147]
Will help people, like me, to see the locked questions, if possible.

**Comment by haoel** [comment]
yes, every src files has the problem description. could you please be more specific ?

**Comment by debodirno** [comment]
I meant, could you please pick up locked questions and solve them with problem descriptions as comments in the beginning?

**Comment by haoel** [comment]
I see. thanks for clarification. 

the locked problem ihasn't public access permission, so it's very hard to retrive the description from the leetcode website, but probably I can get from the thrid party.

If you have any good idea, please let me know.
